### PR TITLE
Tune and remove unused features to reduce memory usage

### PR DESCRIPTION
**Pull request by tiopex**
**old kernel**: Memory: 23400K/32768K available (5201K kernel code, 557K rwdata, 1392K rodata, 1024K init, 232K bss, 9368K reserved, 0K cma-reserved, 0K highmem)

**new kernel**: Memory: 25760K/32768K available (3906K kernel code, 335K rwdata, 932K rodata, 1024K init, 148K bss, 7008K reserved, 0K cma-reserved)

### CODE REVIEW

File: arch/arm/configs/miyoo_defconfig [hunk 1]
```diff
@@ -80,11 +83,6 @@ CONFIG_SND_SOC=y
 CONFIG_SND_SUNIV_CODEC=y
 CONFIG_USB_DYNAMIC_MINORS=y
 CONFIG_USB_OTG=y
-CONFIG_USB_EHCI_HCD=y
-CONFIG_USB_EHCI_HCD_PLATFORM=y
-CONFIG_USB_OHCI_HCD=y
-CONFIG_USB_OHCI_HCD_PLATFORM=y
-CONFIG_USB_STORAGE=y
```
Comment (Apaczer) [inline arch/arm/configs/miyoo_defconfig:87]: we could add support for reading /media at some point?

File: arch/arm/configs/miyoo_defconfig [hunk 2]
```diff
@@ -1,14 +1,19 @@
 CONFIG_LOCALVERSION="-miyoo"
 # CONFIG_LOCALVERSION_AUTO is not set
 CONFIG_DEFAULT_HOSTNAME="miyoo"
-CONFIG_CGROUPS=y
+CONFIG_LOG_BUF_SHIFT=16
+CONFIG_PRINTK_SAFE_LOG_BUF_SHIFT=12
+CONFIG_CC_OPTIMIZE_FOR_SIZE=y
+# CONFIG_MULTIUSER is not set
+# CONFIG_KALLSYMS is not set
+CONFIG_EMBEDDED=y
```
Comment (Apaczer) [inline arch/arm/configs/miyoo_defconfig:9]: what stuff is being cut with this opt enabled? I couldn't read much from descrip. and in src

File: arch/arm/configs/miyoo_defconfig (hunk 1 above)
Comment (tiopex) [reply arch/arm/configs/miyoo_defconfig:87]: This driver is to connect usb drive to handheld, not to share

File: arch/arm/configs/miyoo_defconfig (hunk 2 above)
Comment (tiopex) [reply arch/arm/configs/miyoo_defconfig:9]: This allow to configure expert parameters. In this case I removed multiuser, as we always use only root

File: arch/arm/configs/miyoo_defconfig (hunk 1 above)
Comment (Apaczer) [reply arch/arm/configs/miyoo_defconfig:87]: yes that I understand, but do you imply we can't read external storage from handheld? I see connected USB drive in `lsdisk` but couldn't mount without arg error.

Comment (Apaczer) [reply arch/arm/configs/miyoo_defconfig:87]: Then why rm this config? I don't follow.

Comment (tiopex) [reply arch/arm/configs/miyoo_defconfig:87]: You said that is not working. Also to free memory, umtprd sometimes crash with oom on old kernel

Comment (Apaczer) [reply arch/arm/configs/miyoo_defconfig:87]: Ok umtprd is mega convenient, thanks for explanation. Lets keep usb ex. drives in a backlog for e.g. swaping files (.e.g. savestates) between different SD cards.

**Pull request merged**